srtcore/buffer.cpp

```diff
@@ -187,7 +181,7 @@ void CSndBuffer::addBuffer(const char* data, int len, int ttl, bool order, uint6
         // [PB_SOLO] - 1 packet per message
 
         s->m_ullSourceTime_us = srctime;
```

Comment:
Shouldn't this SourceTime be also of type `time_point`? Note that this is a time INTERNALLY calculated.